src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -94,12 +94,12 @@ bool DoorLockServer::SetActuatorState(chip::EndpointId endpointId, bool newActua
     return false;
```

Comment:
Not at all your PR. But I have to admit that I'm confused by some of those methods returning `true`, others returning `false` and one of them returning `status`...

Reply:
I think it's still in the early stage of being developed....